src/include/error.h

```diff
@@ -147,7 +147,9 @@
         _offset = 0;                                                                               \
         WT_IGNORE_RET(                                                                             \
           __wt_snprintf_len_set(buf, len, &_offset, "WiredTiger assertion failed: '%s'. ", #exp)); \
-        WT_IGNORE_RET(__wt_snprintf(buf + _offset, len - _offset, __VA_ARGS__));                   \
+        /* If we would overflow, finish with what we have. */                                      \
+        if (_offset <= len)                                                                        \
```

Comment:
It might make sense for this condition to be just `_offset < len`; otherwise we may be calling `__wt_snprintf` with buffer length 0. But it doesn't seem like `snprintf` is bothered by 0, so it's probably okay!